docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -45,6 +45,9 @@ To use Iceberg, you need:
 Configuration
 -------------
 
+To configure the Iceberg connector, create a catalog properties file in
+``etc/catalog`` named, for example, ``iceberg.properties``, to mount the
+Iceberg connector as the ``iceberg`` catalog.
```

Comment:
@mosabua Can you the latest consensus about this boilerplate?

Reply:
we are rejigging all those to example.properties .. and wont talk about "mounting" .. @sheajamba is currently working on that so if @deadwind4 wants to update this PR they can work together on wording. I can also chat with @sheajamba if desired.

Alternatively you could close the PR and assume we are going to take care of it soon